core/services/keystore/keys/ocr2key/offchain_keyring.go

```diff
@@ -47,10 +47,12 @@ func newOffchainKeyring(encryptionMaterial, signingMaterial io.Reader) (*Offchai
 	return ok, nil
 }
 
+// OffchainSign signs message using private key
 func (ok *OffchainKeyring) OffchainSign(msg []byte) (signature []byte, err error) {
 	return ed25519.Sign(ed25519.PrivateKey(ok.signingKey), msg), nil
 }
 
+// ConfigDiffieHellman TODO
```

Comment:
Needs comment

Reply:
:shrug: https://en.wikipedia.org/wiki/Diffie%E2%80%93Hellman_key_exchange

Reply:
Borrowed it from another function w/ the same name 👍